### DISCUSSION TITLE
How to??? Connect to PLC Delta !!!

### DISCUSSION THREAD

**Discussion by davoud-azari** [artisan-roaster-scope/artisan#1761] [Q&A]
Which Delta PLC models does the Artisan software connect to? (DVPSA or DVPSS2 or etc.)
And what settings should be made in the software?
Or which machine settings can be used in the Config tab?

**Comment by poundy** [comment]
Have you taken the time to read the documentation? 

https://artisan-scope.org/devices/pid/

You'll likely need https://artisan-scope.org/devices/modbus/ as well
